sharing_portal/views.py

```diff
@@ -60,6 +60,9 @@ def wrapper(request, *args, **kwargs):
 
 def check_view_permission(func):
     def can_view(request, artifact):
+        if artifact.deleted:
```

Comment:
If deleted artifacts are filtered out at query time (which I think is the right way to do it), is this necessary?

Reply:
@diurnalist Yes, as without this check, you could still view the page/launch an artifact that has been deleted.